game/static/game/css/500.css

```
@@ -0,0 +1,156 @@
+    border-color: rgba(240, 192, 64, 0.22);
+}
+
+.crest {
+    width: 72px;
+    height: auto;
+    margin-bottom: 0.85rem;
+    filter: brightness(1.5) contrast(1.2) drop-shadow(0 0 8px rgba(240, 192, 64, 0.28));
+}
+
+.code {
+    font-family: 'Cinzel', serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)